ansible/files/prosody.cfg.lua

```diff
@@ -203,6 +207,9 @@ if ENV_SNIKKET_TWEAK_TURNSERVER ~= "0" or ENV_SNIKKET_TWEAK_TURNSERVER_DOMAIN th
 	turncredentials_secret = ENV_SNIKKET_TWEAK_TURNSERVER_SECRET or assert(io.open("/snikket/prosody/turn-auth-secret-v2")):read("*l");
 end
 
+-- Allow restricted users access to push notification servers
+isolate_except_domains = { "push.snikket.net", "push-ios.snikket.net" }
```

Comment:
This probably means that restricted users should be documented to only support Snikket Android and Snikket iOS apps, anything else (mobile) won't work properly with their account.

Reply:
Replied in the chat, but just for completeness: this line possibly isn't necessary, as mod_isolate_host will not block stanzas originating from the server (e.g. mod_cloud_notify), though it may block direct communication between the app and the push server if that happens over XMPP.

We may in the future switch to a mod_firewall-based solution, which gives more fine-grained control over what is blocked.